classes/simplemap.php

```diff
@@ -732,25 +732,25 @@ function google_map_js_script() {
 			var units                    = '<?php echo esc_js( $options['units'] ); ?>';
 			var limit                    = '<?php echo esc_js( $options['results_limit'] ); ?>';
 			var plugin_url                = '<?php echo esc_js( SIMPLEMAP_URL ); ?>';
-			var visit_website_text        = '<?php echo apply_filters( 'sm-visit-website-text', __( 'Visit Website', 'simplemap' ) ); ?>';
-			var get_directions_text        = '<?php echo apply_filters( 'sm-get-directions-text', __( 'Get Directions', 'simplemap' ) ); ?>';
-			var location_tab_text        = '<?php echo apply_filters( 'sm-location-text', __( 'Location', 'simplemap' ) ); ?>';
-			var description_tab_text    = '<?php echo apply_filters( 'sm-description-text', __( 'Description', 'simplemap' ) ); ?>';
-			var phone_text                = '<?php echo apply_filters( 'sm-phone-text', __( 'Phone', 'simplemap' ) ); ?>';
-			var fax_text                = '<?php echo apply_filters( 'sm-fax-text', __( 'Fax', 'simplemap' ) ); ?>';
-			var email_text                = '<?php echo apply_filters( 'sm-email-text', __( 'Email', 'simplemap' ) ); ?>';
+			var visit_website_text        = '<?php echo apply_filters( 'sm-visit-website-text', __( 'Visit Website', 'SimpleMap' ) ); ?>';
+			var get_directions_text        = '<?php echo apply_filters( 'sm-get-directions-text', __( 'Get Directions', 'SimpleMap' ) ); ?>';
+			var location_tab_text        = '<?php echo apply_filters( 'sm-location-text', __( 'Location', 'SimpleMap' ) ); ?>';
+			var description_tab_text    = '<?php echo apply_filters( 'sm-description-text', __( 'Description', 'SimpleMap' ) ); ?>';
+			var phone_text                = '<?php echo apply_filters( 'sm-phone-text', __( 'Phone', 'SimpleMap' ) ); ?>';
+			var fax_text                = '<?php echo apply_filters( 'sm-fax-text', __( 'Fax', 'SimpleMap' ) ); ?>';
+			var email_text                = '<?php echo apply_filters( 'sm-email-text', __( 'Email', 'SimpleMap' ) ); ?>';
 
 			var taxonomy_text = {};
 			<?php
 			if ( $taxonomies = $this->get_sm_taxonomies( 'array', '', true, 'object' ) ) {
 				foreach ( $taxonomies as $taxonomy ) {
 					?>
-					taxonomy_text.<?php echo $taxonomy->name; ?> = '<?php echo apply_filters( $taxonomy->name . '-text', __( $taxonomy->labels->name, 'simplemap' ) ); ?>';
+					taxonomy_text.<?php echo $taxonomy->name; ?> = '<?php echo apply_filters( $taxonomy->name . '-text', __( $taxonomy->labels->name, 'SimpleMap' ) ); ?>';
 					<?php
 				}
 			}
 			?>
-			var noresults_text            = '<?php echo apply_filters( 'sm-no-results-found-text', __( 'No results found.', 'simplemap' ) ); ?>';
+			var noresults_text            = '<?php echo apply_filters( 'sm-no-results-found-text', __( 'No results found.', 'SimpleMap' ) ); ?>';
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 145 characters

<!-- Comment by [SideCI](https://sideci.com) -->